src/contracts/GPv2Settlement.sol

```diff
@@ -108,9 +110,14 @@ contract GPv2Settlement {
         bytes calldata encodedOrders0,
         bytes calldata encodedOrders1
     ) external {
+        IUniswapV2Pair pair = uniswapPairAddress(token0, token1);
+        uint256 nonce = fetchIncrementNonce(pair);
```

Comment:
Should the lowest nonce for each pair be 1, since in the spec we reserved signatures on nonce 0 to denote "no replay protection"?

Reply:
Oh right!!